app/models/krikri/activity.rb

```diff
@@ -83,5 +83,23 @@ def rdf_subject
       RDF::URI(Krikri::Settings['marmotta']['ldp']) /
         Krikri::Settings['prov']['activity'] / id.to_s
     end
+
+    ##
+    # Return an Enumerator of URI strings of entities (e.g. aggregations or
+    # original records) that pertain to this activity
+    #
+    # @return [Enumerator] URI strings
+    def generated_entity_uris
+      activity_uri = RDF::URI(rdf_subject)  # This activity's LDP URI
+      query = Krikri::ProvenanceQueryClient.find_by_activity(activity_uri)
+      query.each_solution.lazy.map do |s|
+        s.record.to_s
```

Comment:
What's the benefit of the `#to_s`? Would it be more flexible to leave these as `RDF::URI`s?